api/unassignedCases/unassingedCases-util.ts

```diff
@@ -1,88 +1,86 @@
-import {CaseHeader, CcdCase, CcdCaseData, CcdColumnConfig, UnAssignedCases} from './interfaces/index'
+import { CaseHeader, CcdCase, CcdCaseData, CcdColumnConfig, UnAssignedCases } from './interfaces'
 import { caseAssignment, caseId, caseTypeStr } from './unassingedCases-constants'
 
 export function getApiPath(ccdPath: string, caseTypeId: string) {
-    return `${ccdPath}${caseAssignment}?ctid=${caseTypeId}&use_case=ORGCASES`
+  return `${ccdPath}${caseAssignment}?ctid=${caseTypeId}&use_case=ORGCASES`
 }
 
 export function mapCcdCases(caseType: string, ccdCase: CcdCase): UnAssignedCases {
-    const idField = '[CASE_REFERENCE]'
-    const columnConfigs: CcdColumnConfig[]  = mapCcdColumnConfigs(ccdCase)
-    const data: any[] = mapCcdData(ccdCase, columnConfigs, caseType)
-    return {
-        columnConfigs,
-        data,
-        idField,
-    }
+  const idField = '[CASE_REFERENCE]'
+  const columnConfigs: CcdColumnConfig[] = mapCcdColumnConfigs(ccdCase)
+  const data: any[] = mapCcdData(ccdCase, columnConfigs, caseType)
+  return {
+    columnConfigs,
+    data,
+    idField,
+  }
 }
 
 function mapCcdData(ccdCase: CcdCase, columnConfigs: CcdColumnConfig[], caseType: string): any[] {
-    const data = Array<any>()
-    ccdCase.cases.forEach(caseData => data.push(onGeneratedRow(caseData, columnConfigs, caseType)))
-    return data
+  const data = Array<any>()
+  ccdCase.cases.forEach(caseData => data.push(onGeneratedRow(caseData, columnConfigs, caseType)))
+  return data
 }
 
 function onGeneratedRow(ccdCaseData: CcdCaseData, columnConfigs: CcdColumnConfig[], caseType: string) {
```

Comment:
_nitpick:_
This has no return type. It's a nitpick because it looks like this was just a reformat.

Reply:
yes correct just reformat

Reply:
for the tslint issue suppose we have a tech dept ticket to handle the semicolons